packages/graphql/tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "script"
   ],
   "compilerOptions": {
+    "lib": ["es2020"],
```

Comment:
Was this changes to accommodate the `#myVar` syntax? Should this be reverted?

Reply:
Yes, not sure we need to revert or not - seems nice to use the latest `es` version in general?